models/category/src/category-draft/presets/sample-data-goodstore/serveware.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LocalizedString,
+  KeyReference,
+} from '@commercetools-test-data/commons';
+import type { TCategoryDraftBuilder, TCategoryDraft } from '../../../types';
+import empty from '../empty';
+import kitchen from './kitchen';
+
+const kitchenDraft = kitchen().build<TCategoryDraft>();
+
+const serveware = (): TCategoryDraftBuilder =>
+  empty()
+    .name(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('Kitchen Collections')
+        ['en-US']('Kitchen Collections')
+        ['de-DE']('Alles zum Servieren')
+    )
+    .slug(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('kitchen-collections')
+        ['en-US']('kitchen-collections')
+        ['de-DE']('kitchen-collections')
+    )
+    .key('serveware')
+    .orderHint('.86')
```

Comment:
Do we know if not prepending these with a `0` a la `0.86` will cause issues? Seems unlikely, but not sure

Reply:
This is a good question - @stephsprinkle @phofmann any insight as to whether the lack of a leading `0` could cause problems?